### PR TITLE
add align parameter

### PR DESCRIPTION
**Pull request by resfandiari**

### CODE REVIEW

File: lib/elegant_notification.dart [hunk 1]
```diff
@@ -205,6 +209,19 @@ class ElegantNotification extends StatefulWidget {
...
+  ///```dart
+  ///{
+  ///top,
+  ///center,
+  ///bottom
+  ///}
+  ///```
+  ///default value `top`
+  ///
+  final NOTIFICATION_ALIGN notificationAlign;
```
Comment (koukibadr) [inline lib/elegant_notification.dart:223]: rename notificationAlign to notificationPosition to clarify the parameter usage

File: lib/resources/arrays.dart [hunk 2]
```diff
@@ -9,3 +9,8 @@ enum ANIMATION {
   fromRight,
   fromTop,
 }
+enum NOTIFICATION_ALIGN {
```
Comment (koukibadr) [inline lib/resources/arrays.dart:12]: rename `NOTIFICATION_ALIGN` to `NOTIFICATION_POSITION`

File: lib/elegant_notification.dart [hunk 3]
```diff
@@ -300,62 +317,72 @@ class _ElegantNotificationState extends State<ElegantNotification>
...
+                ),
+            ],
           ),
         ),
-      ],
+      ),
     );
   }
 
+  Alignment _getNotificationAlign() {
```
Comment (koukibadr) [inline lib/elegant_notification.dart:373]: rename  `_getNotificationAlign` to `_getNotificationPosition`

**Comment by koukibadr** [comment]
@resfandiari I tested it works perfectly but I added some comments on the pull request changes

**Comment by resfandiari** [comment]
> @resfandiari I tested it works perfectly but I added some comments on the pull request changes

Your changes are fine, I checked them

**Comment by koukibadr** [comment]
> > @resfandiari I tested it works perfectly but I added some comments on the pull request changes
> 
> Your changes are fine, I checked them

okay I will merge and I will rename the parameter after

**Pull request merged**